modules/asset-manager-impl/src/main/java/org/opencastproject/assetmanager/impl/OsgiAssetManager.java

```diff
@@ -248,30 +263,37 @@ public AQueryBuilder createQuery() {
   // OSGi depedency injection
   //
 
+  @Reference(name = "entityManagerFactory", target = "(osgi.unit.name=org.opencastproject.assetmanager.impl)")
   public void setEntityManagerFactory(EntityManagerFactory emf) {
     this.emf = emf;
   }
 
+  @Reference(name = "securityService")
   public void setSecurityService(SecurityService securityService) {
     this.secSvc = securityService;
   }
 
+  @Reference(name = "authSvc")
```

Comment:
Just wondering, are these specific – and not necessarily well chosen – names actually used somewhere or would this work the same with just a bare `@Reference`?

Reply:
I think the names aren't used and a bare `@Reference` would work the same, but I'm not sure, so I just copied them.

Reply:
Fair enough. We can clean that out later.